.github/workflows/main.yml

```diff
@@ -70,6 +70,27 @@ jobs:
       - name: Use latest RuboCop from `master`
         run: |
           echo "gem 'rubocop', github: 'rubocop/rubocop'" > Gemfile.local
+          cat Gemfile.local
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "3.3"
+          bundler-cache: true
+      - run: NO_COVERAGE=true bundle exec rake ${{ matrix.task }}
+
+  oldest-rubocop:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        task:
+          - spec
+    name: "Oldest RuboCop: ${{ matrix.task }}"
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use oldest RuboCop allowed by gemspec
+        run: |
+          sed -nr "s/spec.add_dependency 'rubocop', '~> ([0-9\.]+)'/gem 'rubocop', '= \1'/p" \
+            rubocop-rspec.gemspec > Gemfile.local
+          cat Gemfile.local
```

Comment:
Do you want to keep this line?

Reply:
Yes actually, I find it useful to have some readable output from most CI steps.

I’m not sure if you noticed, but I added a similar `cat` to the edge-rubocop job above. It may not be necessary there, but here in line 91–92 I think it may be hard to understand exactly _what_ is being written into Gemfile.local.